docs/02-app/02-api-reference/04-functions/generate-metadata.mdx

```diff
@@ -497,6 +497,16 @@ export const metadata = {
     ],
     locale: 'en_US',
     type: 'website',
+    videos: [
```

Comment:
I think we should just keep it similar to the one's above (e.g., no need to add a mention that it can just a string).